lib/logger/lib/logger/error_handler.ex

```diff
@@ -141,8 +141,10 @@ defmodule Logger.ErrorHandler do
   end
 
   defp translate([], _min_level, _level, :format, {format, args}, truncate) do
-    {format, args} = Logger.Utils.inspect(format, args, truncate)
-    {:ok, :io_lib.format(format, args)}
+    {:ok,
+     format
```

Comment:
Please assign the result of the pipeline to a var and return `{:ok, var}`.

Reply:
Fine!